packages/sources/uniswap-v3/README.md

```diff
@@ -0,0 +1,71 @@
+# Chainlink External Adapter for Uniswap V3
+
+This adapter allows querying Uniswap V3 contracts
+
+### Environment Variables
+
+| Required? |        Name        |                                                 Description                                                 | Options |                 Defaults to                  |
+| :-------: | :----------------: | :---------------------------------------------------------------------------------------------------------: | :-----: | :------------------------------------------: |
+|    ✅     |      RPC_URL       |               An http(s) RPC URL to a blockchain node that can read the Uniswap V3 contracts                |         |                                              |
+|           |  QUOTER_CONTRACT   | The address of the Uniswap V3 address quoter contract. NOTE: THIS SHOULD NOT BE CHANGED ON ETHEREUM MAINNET |         | `0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6` |
+|           |        FEE         |                                          The Uniswap V3 fee amount                                          |         |                    `500`                     |
```

Comment:
To my knowledge, isn't the fee based on the pool? In other words, should we make this a request parameter instead? I guess we can leave the DEFAULT_FEE as an env var.

Reply:
Mostly sounds good. However we should take into account that the default fee pools (0.05%, 0.3%, 1%) don't all exist for every pool - they must each be created: https://github.com/Uniswap/v3-core/blob/main/contracts/UniswapV3Factory.sol#L35
Example, this token only has 2 fee tiers: https://info.uniswap.org/#/tokens/0x0ace32f6e87ac1457a5385f8eb0208f37263b415

Reply:
@drbh I don't see any iteration of the fee tiers in the commits (nor the configuration of the different tiers as discussed above). Are you sure it got pushed up?